src/controllers/users.js

```diff
@@ -188,6 +190,61 @@ class UserController {
       res
     );
   }
+
+  /**
+   * @name updateUserInfo
+   * @description Updates user profile to complete registration
+   * @param {object} req The request object
+   * @param {object} res The response object
+   * @returns {object} The API response
+   */
+  async updateUserInfo(req, res) {
+    try {
+      const userData = { ...req.body };
+      const { user } = res.locals;
+
+      userData.password = hash.generateSync(userData.password);
+      const data = await UserServices.updateUserInfoByEmail({ ...userData }, user.email);
+
+      if (data === 'own_manage') {
+        return Responses.handleError(409, 'You cannot manage your self', res);
+      } else {
```

Comment:
No need for this else statement. You can simple return the success handler outside the `if` block.

Reply:
You're right, let me change it that way.